packages/xml-lsp-server/src/serverHelper.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  TextDocument,
+  Diagnostic,
+  DiagnosticSeverity,
+  Range
+} from "vscode-languageserver";
+
+import { ILexingError, IRecognitionException } from "chevrotain";
+import { parse } from "@xml-tools/parser";
+
+export const SYNTAX_ERROR_MSG = "Syntax error";
+
+const lexingErrorToDiagnostic = (document: TextDocument) => (
```

Comment:
`lexingErrorToDiagnostic` is a function, declare it directly as such
no need to assign a function Object to a const variable.

Reply:
It's a map method, that maps a nested object, what's wrong with it being declared in this way?

Reply:
see above